### ISSUE TITLE
Error in Symfony 4

### ISSUE THREAD

**Issue by gabiudrescu** [Orbitale/CmsBundle#12]
In https://github.com/Orbitale/CmsBundle/blob/3.x/Controller/CategoryController.php there is the line below causing trouble in Symfony4: 

```php
        $categories = $this->get('orbitale_cms.category_repository')->findFrontCategories($slugsArray);
```

It should either be:

```php
        $categories = $this->getDoctrine()->getRepository('App\\Entity\\Category')->findFrontCategories($slugsArray);
```

Or the definition of the service must be set to public, as it is default private.

**Comment by hermann8u** [comment]
Good point for you! I have the same error and this issue isn't present only in the CategoryController but also in PageController. I will add an other thing to your review:
Instead of ->getRepository('App\\Entity\\Category'), we could get the full namespace of the classes from the config

I wil try to make a pull request to solve this issue. I already solve the problem locally

**Comment by Pierstoval** [comment]
Fixed by #14 and released new v3.0.6 tag 👍
